bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZZTo2L2Q01j_5f_NLO_FXFX/ZZTo2L2Q01j_5f_NLO_FXFX_madspin_card.dat

```diff
@@ -0,0 +1,33 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
```

Comment:
please cleanup the datacards following these instructions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T